lib/helpers.ts

```diff
@@ -84,6 +84,8 @@ export interface Entity {
     WPAtlas: WPAtlas;
     biospecimen: Entity[];
     diagnosis: Entity[];
+    demographics: Entity[];
+    cases: Entity[];
```

Comment:
There is some duplication here. `cases` is a field where we merge diagnosis and demographics together.

Reply:
one way to handle this would be in typing, e.g. you could type it as `cases:Pick<Entity, whatevermembersarerelevant>[]`, etc. that would make it clearer what each is used for even if there's overlap

Reply:
Oh I meant the data itself is now duplicated. cases is basically "diagnosis + demographics". Maybe okay for now. It is shallow copy though, so maybe okay to duplicate.

Reply:
how about a utility method `getCases(Entity)` -> `demographics.concat(diagnosis)` instead of the new member?

Reply:
We cannot simply concat them, we have to merge actual objects by htan participant id. The utility function for that one is here: https://github.com/ncihtan/htan-portal/blob/master/lib/helpers.ts#L238